modules/lib/src/main/resources/assets/js/app/saga/event/outgoing/EnonicAiToggleDialogEvent.ts

```diff
@@ -0,0 +1,27 @@
+import {Event} from '@enonic/lib-admin-ui/event/Event';
+import {ClassHelper} from '@enonic/lib-admin-ui/ClassHelper';
+
+export class EnonicAiToggleDialogEvent
```

Comment:
Shouldn't it be "Open/Show", not "Toggle". Will it close the dialog as well? Not sure it's expected behaviour to close the Assistant from HtmlArea

Reply:
Agree, Iet's make it just open the dialog